Poker.BE/Poker.BE.Domain/Game/Round.cs

```diff
@@ -25,69 +26,256 @@ public enum Move
         #region fields
         private ICollection<Player> activeUnfoldedPlayers;
         private Turn currentTurn;
-        private Player dealer;
+        //private Player dealer;        Is dealer needed??
```

Comment:
i didn't understand if fully, but as long as you understand it and it make sense i'm cool with it 👍